src/qml/FeatureForm.qml

```diff
@@ -76,86 +76,92 @@ Page {
     id: container
     anchors.fill: parent
 
-    Flickable {
-      id: flickable
+    ListView {
+      id: tabBarListView
       Layout.fillWidth: true
-      Layout.preferredHeight: tabRow.height
+      Layout.preferredHeight: 48
+      orientation: Qt.Horizontal
+      visible: form.model.hasTabs
+      model: form.model.hasTabs ? form.model : 0
+      highlightFollowsCurrentItem: true
+      highlight: Item {
+        Rectangle {
+          height: 2
+          color: Theme.mainColor
+          radius: 4
+          width: parent.width
+          anchors.bottom: parent.bottom
+        }
+      }
```

Comment:
If we go for this fix, we will need to implement it as a QfTabBar item, and replace our uses of TabBar {} with that across the board. If we don't do that, we'll end up with inconsistent tab bars.